Geometry.cs

```diff
@@ -23,6 +23,15 @@ namespace GrayBMP;
 /// <summary>A point in 2D space, with double-precision coordinates (X, Y)</summary>
 readonly record struct Point2 (double X, double Y) {
    public (int X, int Y) Round () => ((int)(X + 0.5), (int)(Y + 0.5));
+
+   /// <summary>Returns a new point for a given point which is radially rotated.</summary>
+   public readonly Point2 RadialMove (double r, double theta) => new (X + r * Math.Cos (theta), Y + r * Math.Sin (theta));
+
+   /// <summary>Returns angle between two points.</summary>
+   public readonly double AngleTo (Point2 b) {
+      double dx = b.X - X, dy = b.Y - Y;
+      return Math.Atan2 (dy, dx);

```

Comment:
Try doing a.AngleTo (a)

Reply:
Radian value will be always zero, so tan inverse computation is not required. Updated the changes.